Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Attachments/under_attachments.yml

```diff
@@ -347,6 +347,7 @@
     whitelist:
       tags:
       - RMCGrenadeM40
+      - RMCGrenadeTraining
```

Comment:
You can drop this tag whitelist if you add grenade m40 tag to the training grenade

Reply:
done, thank you